uxarray/grid/intersections.py

```diff
@@ -11,7 +11,7 @@
 
 
 @njit(parallel=True, nogil=True, cache=True)
-def fast_constant_lat_intersections(lat, edge_node_z, n_edge):
+def constant_lat_intersections_edges(lat, edge_node_z, n_edge):
     """Determine which edges intersect a constant line of latitude on a sphere.
```

Comment:
Please include a notice that due to the curvature of great circle arcs, such edges selections are not  accurate or correct, it only serves as approximation for performance purpose

Reply:
I have named the function `constant_lat_intersections_no_extreme` and updated the docstrings to take a note of this.

Reply:
sounds good